packages/open-api-gateway/src/project/smithy/smithy-build-project.ts

```diff
@@ -3,21 +3,19 @@ SPDX-License-Identifier: Apache-2.0 */
 import * as path from "path";
 import { Project, ProjectOptions, SampleFile } from "projen";
 import { SmithyBuild } from "projen/lib/smithy/smithy-build";
+import { SampleExecutable } from "./components/sample-executable";
 import { SmithyGeneratedOutput } from "./components/smithy-generated-output";
 import { SmithyBuildOptions } from "./types";
+import { SmithyServiceName } from "../types";
 
 /**
  * Options for a smithy build project
  */
 export interface SmithyBuildProjectOptions extends ProjectOptions {
   /**
-   * Fully qualified service name
+   * Smithy service name
    */
-  readonly fullyQualifiedServiceName: string;
-  /**
-   * Absolute path to the model directory
-   */
-  readonly modelPath: string;
+  readonly serviceName: SmithyServiceName;
```

Comment:
This is a breaking change as well right? Given that these options are public interface for the project?

Reply:
This project is internal so I think this is ok! Though can technically get at it via `import { SmithyBuildProject } from "@aws-prototyping-sdk/open-api-gateway/lib/project/smithy/smithy-build-project";`...